src/components/missions/MissionsList.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { Table } from 'react-bootstrap';
+import { useDispatch, useSelector } from 'react-redux';
+import MissionItem from './MissionItem';
+import getMissionsFromApi from '../../redux/missions/missions-action';
+
+const MissionsList = () => {
+  const missionsData = useSelector((state) => state.missions.missions);
```

Comment:
I think there are too many missions here. You should change it to make it more readable.

Reply:
Hi @Leboroz, thanks. 
I agree 100%., I gonna implement it. 👌